packages/react-next/RELEASE_NOTES.md

```diff
@@ -215,6 +215,10 @@ TODO: document any API or functionality changes
 
 - Updated enums to string union type: `PivotLinkFormat`, `PivotLinkSize`. (#13370)
 
+## New features
+
+- Pivot overflow (TODO: Add more detailed documentation)
```

Comment:
```suggestion
- Pivot supports displaying an overflow menu when there is not enough room to display all of the tabs. This can be enabled by setting `overflowBehavior="menu"` on the Pivot.
```